src/components/side-menu/side-menu-stats.html

```diff
@@ -0,0 +1,14 @@
+<div class="stats">
+  <div class="stats-unit">
+    <span id="stat-songs">328</span>
+    <span>Songs</span>
+  </div>
+  <div class="stats-unit">
+    <span id="stat-albums">52</span>
+    <span>Albums</span>
+  </div>
+  <div class="stats-unit">
+    <span id="stat-artists">87</span>
+    <span>Artists</span>
+  </div>
+</div>
```

Comment:
I think this could be a little more semantic. I know it might not feel like semantics are important here, but it looks like a list to me. Although, you _do_ have another list underneath this on the page... 🤔 
https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i?utm_source=mybridge&utm_medium=blog&utm_campaign=read_more

Reply:
yes, absolut right, i hate that too, will rebuild it, thanks

Reply:
solved